wpilibj/src/athena/cpp/lib/HALUtil.cpp

```diff
@@ -78,7 +80,11 @@ void ThrowHalHandleException(JNIEnv *env, int32_t status) {
   halHandleExCls.Throw(env, buf.c_str());
 }
 
-constexpr const char wpilibjPrefix[] = "edu.wpi.first.wpilibj";
+#ifndef _WIN32
+constexpr const char JNI_wpilibjPrefix[] = "edu.wpi.first.wpilibj";
+#else
+extern const char JNI_wpilibjPrefix[] = "edu.wpi.first.wpilibj";
```

Comment:
This was what was required to get MSVC 2015 working with this. It was giving me errors saying an array with internal linkage wouldn't work.